### PR TITLE
fixing the aws periodic job and adding a presubmit job

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**

### CODE REVIEW

File: config/jobs/cert-manager/cert-manager-periodics.yaml [hunk 1]
```diff
@@ -512,7 +512,7 @@ periodics:
     containers:
     - image: eu.gcr.io/jetstack-build-infra-images/golang-aws@sha256:1f330e4c9552ca383d157067b73fb0e090b64b0777939fd59e58b60e06020d66
       args:
-      - -c
+      - bash -c
```
Comment (jakexks) [inline config/jobs/cert-manager/cert-manager-periodics.yaml:515]: bash and -c have to be separate times in the list
```suggestion
      - bash
      - -c
```

**Review by jakexks — CHANGES_REQUESTED** [review]

**Comment by RinkiyaKeDad** [comment]
@jakexks made the changes, PTAL :)

**Review by jakexks — APPROVED** [review]
/ok-to-test

**Pull request merged**